test/end-to-end/pages/screeners/mentalcapacity.js

```diff
@@ -1,19 +1,23 @@
 'use strict';
 
-const commonContent = require('app/resources/en/translation/common');
-const content = require('app/resources/en/translation/screeners/mentalcapacity');
+const contentEn = require('app/resources/en/translation/common');
+const contentCy = require('app/resources/cy/translation/common');
+const mentalCapacityEn = require('app/resources/en/translation/screeners/mentalcapacity');
+const mentalCapacityCy = require('app/resources/cy/translation/screeners/mentalcapacity');
 
-module.exports = async function(answer) {
+module.exports = async function(language ='en', answer) {
     const I = this;
+    const commonContent = language === 'en' ? contentEn : contentCy;
+    const mentalCapacityContent = language === 'en' ? mentalCapacityEn : mentalCapacityCy;
 
     await I.checkPageUrl('app/steps/ui/screeners/mentalcapacity');
-    await I.waitForText(content.question);
-    await I.see(content.hintText1);
-    await I.see('You can read about');
-
+    if (language === 'en') {
+        await I.waitForText(mentalCapacityContent.question);
+        await I.see(mentalCapacityContent.hintText1);
+        await I.see('You can read about');
+    }
```

Comment:
again, not sure why you'd only want to check the text here on English journeys? It will likely help with test stability for both languages